### PR TITLE
Add wandb

### PR DESCRIPTION
**Pull request by scottire**
Enables:
```python
linerider_from_wandb(project='pytorch-sweeps-demo', entity='_scott', filters={"display_name": {"$in": ["ethereal-sweep-5"]}})
```

### CODE REVIEW

File: src/lossrider/lossridersrc.py [hunk 1]
```diff
@@ -400,3 +400,59 @@ def lossrider(
...
+        x (str): The metric to use for the x-axis. Defaults to "_step".
+        y (str): The metric to use for the y-axis. Defaults to "loss".
+        hue (str, optional): The column to use for differentiating lines. Defaults to "run.name".
+        outfile (str): The name of the output file. Defaults to "lossrider_wandb.save".
+        **kwargs: Additional keyword arguments to pass to the lossrider function.
+
+    Returns:
+        None
+    """
+    api = wandb.Api()
```
Comment (jndean) [inline src/lossrider/lossridersrc.py:432]:
```suggestion
    import wandb
    api = wandb.Api()
```

File: src/lossrider/lossridersrc.py [hunk 2]
```diff
@@ -400,3 +400,59 @@ def lossrider(
         return IFrame("https://www.linerider.com/", 900, 600)
     except ImportError:
         pass
+
+import wandb
+import pandas as pd
+from typing import List, Optional, Dict, Any
```
Comment (jndean) [inline src/lossrider/lossridersrc.py]: Pls move these to the top of the file and omit wandb, I'd prefer not to add that as a dependency for all "users".
If we're being honest pandas should already have been a dependency so I'll accept that addition.

**Review by jndean — CHANGES_REQUESTED** [review]
Excellent value add 🙏 Minor changes requested

**Comment by scottire** [comment]
I've added those changes. 🛷

**Review by jndean — APPROVED** [review]
🎿

**Pull request merged**